src/sagemaker/estimator.py

```diff
@@ -1061,7 +1062,7 @@ def transformer(self, instance_count, instance_type, strategy=None, assemble_wit
             container_def = model.prepare_container_def(instance_type)
             model_name = model.name or name_from_image(container_def['Image'])
             vpc_config = model.vpc_config
-            self.sagemaker_session.create_model(model_name, role, container_def, vpc_config)
+            self.sagemaker_session.create_model(model_name, role, container_def, vpc_config, tags=tags)
```

Comment:
how about defaulting to `self.tags` if `tags` is `None`? I think both `transformer()` implementations should have the same behavior around which tags are propagated.